portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,34 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Akhil Binani's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Akhil Binani's Portfolio</h1>
+      <img id = "My-pic" src="image/image.jpeg" >
+      <div id = "Bio">
+        <p>
+        <h2> Hello there </h2>
+        <h3> Short Description of myself </h3>
+        <br>
+        I am a gamer/coder/Dj. I love to do competetive programming or competetive CSGO, these are the two compi of my life.
+        <br>I am a CSE student IIT(ISM) Dhanbad. 
+        <br>I am  currently an SDE intern at AWS Berlin.
+        <br>If you want to know more about me then visit my profiles below.
+
+        <br><a href = "https://codeforces.com/profile/Sad_reacts_only" > Visit My Codeforces profile </a>
+        <br><a href = "https://www.codechef.com/users/bin_0808" > Visit My Codechef profile </a>
+        <br><a href = "https://www.linkedin.com/in/akhil-binani/" > Visit My LinkedIn profile </a>
+        </p>
+      </div>
+    <div id=project>
```

Comment:
Please change this id to class and keep element names (i.e. class name) in a string.